cli/status.go

```diff
@@ -0,0 +1,214 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/exercism/cli/user"
+)
+
+// Status represents the results of a CLI self test.
+type Status struct {
```

Comment:
All of this stuff in `Status` builds up to using the `text/template` package for formatting. The template used is defined as a constant at the bottom of this file.